build.sbt

```diff
@@ -36,22 +35,13 @@ def common: Seq[Setting[_]] = releaseSettings ++ bintraySettings ++ evictionSett
   licenses := Seq(("Apache-2.0", url("http://www.apache.org/licenses/LICENSE-2.0.html"))),
   homepage := Some(url("https://www.lagomframework.com/")),
   sonatypeProfileName := "com.lightbend",
-  headers := headers.value ++ Map(
-     "scala" -> (
-       HeaderPattern.cStyleBlockComment,
-       """|/*
-          | * Copyright (C) 2016-2018 Lightbend Inc. <https://www.lightbend.com>
-          | */
-          |""".stripMargin
-     ),
-     "java" -> (
-       HeaderPattern.cStyleBlockComment,
-       """|/*
-          | * Copyright (C) 2016-2018 Lightbend Inc. <https://www.lightbend.com>
-          | */
-          |""".stripMargin
-     )
-  ),
+  headerLicense := Some(HeaderLicense.Custom(
+    "Copyright (C) 2016-2018 Lightbend Inc. <https://www.lightbend.com>"
+  )),
+  headerMappings := headerMappings.value ++ Seq(
+    HeaderFileType.scala,
+    HeaderFileType.java
+  ).map(_ -> HeaderCommentStyle.cStyleBlockComment).toMap,
```

Comment:
I think you can take the `headerMappings` setting out, it looks the same as the default:

https://github.com/sbt/sbt-header/blob/707e2fe57d5f7b13c84b39e16173d6349813b1fc/src/main/scala/de/heikoseeberger/sbtheader/HeaderPlugin.scala#L126-L129

Reply:
:+1: I've seen this cargo culting before.